client/src/Organisms/BurgerMenu/BurgerMenuStyled.js

```diff
@@ -25,9 +25,10 @@ const BurgerMenuStyled = styled.div`
     grid-area: 3 / 1 / 4 / 2;
     transform: none;
     transition: none;
+    display: none;
   }
   @media only screen and (min-width: 1600px) {
-    grid-area: 3 / 2 / 4 / 3;
+    display: none;
```

Comment:
This media query is not neccessary anymore. If you add display none from 800 up, you dont need anythin else.

Reply:
True. Fixed.